src/components/activity/ActivityNotification.tsx

```diff
@@ -16,6 +17,7 @@ interface Props {
 export const ActivityNotification: FC<Props> = ({ notification, close }) => {
   const titleId = useId();
   const { activity } = notification;
+  const { dismissAlert } = useNotifications();
```

Comment:
From what I understood, it should close the notification panel, not the notification itself.